src/app/GitCommands/Git/Status.cs

```diff
@@ -1,7 +0,0 @@
-﻿namespace System
-{
-    internal static partial class NativeMethods
-    {
-        internal const int STATUS_CONTROL_C_EXIT = -1073741510; // 0xC000013A
```

Comment:
Why move to a different DLL?

Reply:
Because `GitCommands` can now see the `NativeMethods` class that is internal to `GitExtUtils`, so it's either rename one of them or merge them.